packages/docs/pages/reference/elements/example.mdx

```diff
@@ -8,12 +8,68 @@ import {
 
 # `<example>`
 
-Short description of what the `<example>` tag does.
+The `<example>`, `problem`, and `question` tags are closely related and have
+the same syntax.  The `title` element is optional but encouraged, and the
+structure can contain `hint`s, `answer`s, and `solution`s (in that order).
```

Comment:
Let's make sure tags are referred to using
```
[`<tag-name>`](tag-name)
```
syntax.

I think this is worth breaking into two paragraphs: the first explains what an example is for. The second explains that other elements are part of the `example-like` class of elements and share the structure of examples, but whose labels will be different.